package.json

```diff
@@ -137,7 +139,7 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "28.1.1",
     "jest-axe": "6.0.0",
-    "jest-environment-jsdom": "28.1.1",
+    "jest-environment-jsdom": "28.1.3",
```

Comment:
Curious to know why this bumped up?

Reply:
I was troubleshooting some unit test weirdness and while I don't think it was required to fix it, seemed like a good opportunity to keep it up to date since it's just a patch and all the tests still pass.